src/pages/workspaces/workspace/jobHistory/CallTable.js

```diff
@@ -346,6 +347,37 @@ const CallTable = ({
             },
             ...(isAzure // GCP workspaces do not have inputs/outputs/logs (only log file is at top level, linked in workflow dashboard)
               ? [
+                  {
+                    size: { basis: 200, grow: 1 },
+                    field: 'cost',
+                    headerRenderer: () =>
+                      h(Sortable, { sort, field: 'cost', onSort: setSort }, [
+                        'Approximate Cost',
+                        h(
+                          TooltipTrigger,
+                          {
+                            content:
+                              'This is a calculated list price and does not include disk cost, subworkflow costs, or any cloud account discounts.',
```

Comment:
Might be clearer to say: "Approximate cost is calculated based on the the list price of the VM used and does not include disk cost or any cloud account discounts."

Reasons:
 * In this table we see the subworkflows and their missing costs, there is no total number they're excluded from
 * The number shown isn't a published list price, the list price is one input to it.